setup-buildengine.sh

```diff
@@ -129,6 +131,27 @@ startVM() {
     sleep 2
 }
 
+stopVM() {
+# shut the VM down cleanly so that it has time to flush its disk
```

Comment:
Missing indentation, extra blank line after the block. `waitc` should be declared local and initialized.

Anyway, at the end this is only called once from where it originally existed literally, so I would prefer reorganizing the commits and leave the original code untouched. There are more temporary changes that deserve rebasing/squashing.

Reply:
you are right - restored the original code